python/hsfs/engine/python.py

```diff
@@ -819,16 +848,9 @@ def acked(err, msg):
                     progress_bar.colour = "RED"
                     raise err  # Stop producing and show error
             # update progress bar for each msg
-            progress_bar.update()
-
-        # initialize progress bar
-        progress_bar = tqdm(
-            total=dataframe.shape[0],
-            bar_format="{desc}: {percentage:.2f}% |{bar}| Rows {n_fmt}/{total_fmt} | "
-            "Elapsed Time: {elapsed} | Remaining Time: {remaining}",
-            desc="Uploading Dataframe",
-            mininterval=1,
-        )
+            if not feature_group._multi_part_insert:
+                progress_bar.update()
```

Comment:
is the user somehow informed of the progress being made for multi-insert?

Reply:
maybe we could write how many entries have been submitted.

Reply:
I guess the right approach would be to keep the progress bar similar to the producer in the feature group object.
But I am not sure how it will look like then and if it works to update it across function calls. This is an edge case mostly done in applications and high performance, not interactively in notebooks, so my thinking was that the progress bar is not important and will just slow it down.